modules/db/src/main/scala/gem/dao/ObservationDao.scala

```diff
@@ -21,6 +21,7 @@ object ObservationDao {
                     ${o.id.index},
                     ${o.title},
                     ${o.instrument})
+       ON CONFLICT DO NOTHING
```

Comment:
I don't think we want to limit the seqexec to a single observation per program, which is what we'd end up with if we delete the entire program every time we fetch an observation.  I basically want to leave the program and observation stub alone if they exist, but create them otherwise.  If a query to check and then create if necessary is done in a single transaction, that should do it?